tests/e2e/demo-components-compare.spec.ts

```diff
@@ -24,13 +24,11 @@ const IGNORED_TESTS: string[] = [
   'Select',
   'Spinner',
   'Stack',
-  'Tag',
   'Text',
   'TextArea',
   'TextField',
   'Toast',
   'Tooltip',
-  'UNSTABLE_Section',
```

Comment:
Why you are removing also the Section component?

I think it is an unrelated change so at least it should be in a separate commit 🙏 Thanks.

Reply:
It uses Tag, so now we can allow testing again. But you are right I moved it to a separate commit.